addons/mail/static/src/owl/components/composer_text_input/composer_text_input.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.component.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
-            <textarea t-ref="textarea"/>
+            <textarea class="o_ComposerTextInput_textarea" t-esc="storeProps.composer.textInputContent" placeholder="Write something..." t-on-keydown="_onKeydownEditable" t-on-input="_onInputEditable" t-ref="textarea" />
```

Comment:
I would rename the events to not contain the "editable" word, maybe replaced with textarea.

Reply:
Done